frontend/pages/policies/ManagePoliciesPage/components/InstallSoftwareModal/InstallSoftwareModal.tsx

```diff
@@ -22,18 +27,18 @@ import Button from "components/buttons/Button";
 import { ISoftwareTitle } from "interfaces/software";
 import TooltipWrapper from "components/TooltipWrapper";
 
-const getPlatformDisplayFromPackageExtension = (ext: string | undefined) => {
+const getPlatformFromExtension = (ext: string | undefined): Platform | null => {
   switch (ext) {
```

Comment:
> I think this’ll break for zip files, which we might use for FMA for Windows. Will follow up in a PR comment. Thinking we should use for software title source instead as that’s more reliable.

@iansltx We are currently showing display text of macOS for zip files in the dropdown

Reply:
Yeah, we can fix _that_ if/when we run across FMAs for Windows (or Linux?) that are delivered by zip file. Mainly trying to avoid writing new brittle code that we have to remember to fix elsewhere, and we know `source` on the software title won't have that issue (though it requires a mapping rather than having an explicit platform field).